spec/schema.json

```diff
@@ -1168,7 +1168,18 @@
       },
       "Country": {
         "type": "string",
-        "enum": [
+        "anyOf": [
```

Comment:
Not sure why this enum has come back in - I thought it wasn't required for just the entity validation. Anyway, no need for it to hold up merging and other work, so will approve.